.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+**Description:** Describe in a couple of sentence what this PR adds
+
+**JIRA:** Link to JIRA ticket
+
+**Reviewers:**
+- [ ] tag reviewer 
+- [ ] tag reviewer 
+
+**Merge checklist:**
+- [ ] All reviewers approved
+- [ ] CI build is green
+- [ ] Commits are squashed
+- [ ] PR author is listed in AUTHORS
```

Comment:
Why do we need an AUTHORS file?  Github maintains this for us.

Reply:
Yes, let's remove that file.

Reply:
Will do.  We should discuss with @jmbowman if we want to clean some of these files out of the cookiecutter.